kernel-util/src/main/java/org/sakaiproject/util/Validator.java

```diff
@@ -359,7 +359,27 @@ public static String limitFormattedText(String value, int length)
         return ret.toString();
     }
 
- 
+	/**
+	Checks whether the email address has got valid local domain
+	@param email
+	 */
+	public static boolean isValidDomain(String email) {
```

Comment:
Sure

On 27 October 2015 at 16:16, Surya Janani Govindarajan <
notifications@github.com> wrote:

> In kernel-util/src/main/java/org/sakaiproject/util/Validator.java
> <https://github.com/ox-it/wl-kernel/pull/75#discussion_r43145011>:
>
> > @@ -359,7 +359,27 @@ public static String limitFormattedText(String value, int length)
> >          return ret.toString();
> >      }
> >
> > -
> > +	/**
> > +	Checks whether the email address has got valid local domain
> > +	@param email
> > +	 */
> > +	public static boolean isValidDomain(String email) {
>
> Can we have it like isAllowedLocalEmailDomain as it checks for official
> local domains
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ox-it/wl-kernel/pull/75/files#r43145011>.
>



-- 
  Matthew Buckett, VLE Developer, IT Services, University of Oxford